prov/tcp/src/xnet_domain.c

```diff
@@ -258,10 +257,10 @@ int xnet_domain_open(struct fid_fabric *fabric_fid, struct fi_info *info,
 	if (ret)
 		goto close;
 
-	if (fabric->progress.auto_progress) {
+	if (domain->progress.auto_progress) {
 		ret = xnet_start_progress(&domain->progress);
 	} else {
-		ret = xnet_add_wait_eq_list(domain);
+		ret = xnet_domain_add_eq_list(domain);
```

Comment:
Isn't this adding the domain to every EQ?  I think we only need to add from xnet_ep_bind() like you have below.